CodingStandards.txt

```diff
@@ -91,8 +88,6 @@ All other entities' first alpha is lower case.
 4. Variable prefixes:
 
 a. Leading underscore "_" to parameter names (both normal and template).
-- Exception: "o_parameterName" when it is used exclusively for output. See 6(f).
```

Comment:
Or, being so rare, not worry about it?  My style is that input parameters should be values or const references, and output parameters should be pointers, and pointers should be prefixed with p (my only prefix rule) so it's obvious when I'm dealing with an output parameter.  And since all but the lowest-level code should avoid raw pointers, output parameters indeed should be rare.